app/views/CreditAndRefunds.scala.html

```diff
@@ -103,13 +104,34 @@ <h2 class="govuk-heading-m">@{balance.get.firstPendingAmountRequested.get.toCurr
             </summary>
             <div class="govuk-details__text">
                 <dl class="govuk-summary-list">
-                    @creditCharges.map { charge =>
-                        <div class="govuk-summary-list__row">
-                            <dt class="govuk-summary-list__value">
-                                <a href="@getControllerHref(charge._1.documentDetail, isAgent)" class="govuk-link">@charge._1.dueDate.get.toLongDate @messages("credit-and-refund.payment")</a>
-                            </dt>
-                            <dd class="govuk-summary-list__value">@charge._1.documentDetail.paymentOrChargeCredit.get.toCurrencyString</dd>
-                        </div>
+                    @for(charge <- creditCharges){
+                        @if(isMFACreditsAndDebitsEnabled){
+                            @if(charge._1.documentDetail.validMFACreditDescription()) {
+                                <div class="govuk-summary-list__row">
+                                    <dt class="govuk-summary-list__value">
+                                        @* => MISUV-3648*@
```

Comment:
Please remove the ticket number from the code.

Reply:
comment removed